frontend/src/pages/distributedWorkloads/global/workloadStatus/DWStatusOverviewDonutChart.tsx

```diff
@@ -13,6 +13,7 @@ export const DWStatusOverviewDonutChart: React.FC = () => {
   const { workloads } = React.useContext(DistributedWorkloadsContext);
 
   const statusCounts = React.useMemo(() => getStatusCounts(workloads.data), [workloads.data]);
+  const statusTypes = Object.keys(statusCounts) as WorkloadStatusType[];
```

Comment:
Why not an enum for WorkloadStatus? You seem to be wanting to do that a lot...

Reply:
Previously I had an `Unknown` status that was only conditionally included in `statusCounts` and wanted to hide it from the chart if it wasn't there, but that has been replaced by falling back to `Pending` so maybe it makes sense to have an enum now. Hmm.

Reply:
ahh -- part of the reason for using Object.keys (which returns `string[]` and needs this assertion) is that in `utils.tsx`, the order of keys in `knownStatusConditions` matters because we want the first matching condition found there, and if we just did `Object.values(Enum)` we'd be making the order of the enum values affect behavior elsewhere which feels fragile. But maybe we need to explicitly just make a `statusConditionsInOrder` array to go through instead of relying on the key order there.

Reply:
Rebased and updated to use an enum here, `Object.values()` around it with no type assertion where we were using `Object.keys` before with a type assertion, and use a `statusesInEvalOrder` to be explicit about that behavior and not rely on the order of keys in the `knownStatusConditions` object.